DuckDuckGo/UserText.swift

```diff
@@ -1263,6 +1265,34 @@ But if you *do* want a peek under the hood, you can find more information about
         return String(format: localized, expiration)
     }
 
+    static func renewingTrialSubscriptionInfo(billingPeriod: PrivacyProSubscription.BillingPeriod, renewalDate: String) -> String {
+        let localized: String
+
+        switch billingPeriod {
+        case .monthly:
+            localized = NSLocalizedString("subscription.subscription.renewing.trial.monthly.caption",
+                                          value: "Your free trial ends on %@ & automatically converts to a monthly paid subscription on that day.",
```

Comment:
Just to confirm if the copy with "&" instead of "and" is the final approved one?

Reply:
So far, it's in copy review. So we'll see.